lib/rules/jsx-fragments.js

```diff
@@ -171,7 +172,7 @@ module.exports = {
       ImportDeclaration(node) {
         if (node.source && node.source.value === 'react') {
           node.specifiers.forEach((spec) => {
-            if (spec.imported && spec.imported.name === fragmentPragma) {
+            if (spec.type === 'ImportSpecifier' && spec.imported && spec.imported.name === fragmentPragma) {
```

Comment:
why is this needed? what are the possible `type` values of the objects contained an ImportDeclaration node's `specifiers` array - is it more than just `ImportSpecifier`?

Reply:
> what are the possible type values of the objects contained an ImportDeclaration node's specifiers array - is it more than just ImportSpecifier?

`ImportSpecifier | ImportDefaultSpecifier | ImportNamespaceSpecifier` .

I added it because only ImportSpecifier has the imported property.

Reply:
would `'imported' in spec` narrow properly?

Reply:
Yes, it will be narrowed down. Alternatively, it might be better to do it as follows.

```js
 if (spec.type === 'ImportSpecifier' && spec.imported.name === fragmentPragma) { ...
```